index.js

```diff
@@ -224,17 +230,24 @@ Slideout.prototype._initTouchEvents = function() {
     if (Math.abs(dif_x) > 20) {
       self._opening = true;
 
-      if (self._opened && dif_x > 0 || !self._opened && dif_x < 0) { return; }
+      if (self._side === 'left') {
```

Comment:
Using orientation we can omit the branching and make clearer code, my proposal is replace the whole `if - else if block` with:

```
var oriented_dif_x = dif_x * this._orientation;
if(self._opened && oriented_dif_x > 0 || !self._opened && oriented_dif_x < 0) { return; }
if(oriented_dif_x <= 0) {
  translateX = dif_x + self._padding * this._orientation;
}
```